src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1669,6 +1670,14 @@ impl Coordinator {
         })
     }
 
+    pub(super) fn sequence_explain_schema(
+        &mut self,
+        ExplainSchemaPlan { json_schema, .. }: ExplainSchemaPlan,
+    ) -> Result<ExecuteResponse, AdapterError> {
+        let jsonb = Jsonb::from_str(&json_schema)?;
+        Ok(Self::send_immediate_rows(vec![jsonb.into_row()]))
+    }
+
```

Comment:
When can this fail?

Reply:
If we ever not store a valid json here, then this might fail.